servers/text/text_server.h

```diff
@@ -582,10 +582,12 @@ class TextServer : public RefCounted {
 	void shaped_text_debug_print(const RID &p_shaped) const;
 #endif
 
+#ifndef DISABLE_DEPRECATED
```

Comment:
`DISABLE_DEPRECATED` should be added to binds as well:

https://github.com/timothyqiu/godot/blob/5f677ae0ae854e5ccce8a3d0af87a42ed0b6c648/servers/text/text_server_extension.cpp#L366-L368